tests/e2e/test_cases_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func getTestCases() ([]e2eTestCase, error) {
 	testCases := getPostgreSQLTestCases()
-	// testCases = append(testCases, rediscacheTestCases...)
+	testCases = append(testCases, rediscacheTestCases...)
```

Comment:
Please make this the _first_ group of test cases added to `testCases`. Redis takes longer than anything else we have to provision and deprovision, so I want to assurance that regardless of how serialized or parallelized the tests may be, the Redis cases are always kicked off first.

Reply:
I have moved redis to the top.